resources/tests/test_resources_mutations.py

```diff
@@ -657,6 +675,16 @@ def test_update_harbor(superuser_api_client, harbor, availability_level, municip
         variables["location"]["coordinates"][1],
     )
     assert get_harbor_media_folder(harbor, image_file_name) in image_file
+
+    assert len(map_files) == 3
+
+    expected_urls = [
+        get_harbor_media_folder(harbor, file_name) for file_name in map_file_names
+    ]
+    # Test that all the expected files are in the instance map files
+    for file in map_files:
+        assert any([expected_url in file["url"] for expected_url in expected_urls])
```

Comment:
you mean `all(...)`?

Reply:
No no. The outer `for` goes through the files on the instance, so that's the one that should match `all()`. The inner `any()` checks that either of the `expected_url`s match `any` of the ones on the instance (hence, `any()`).

Reply:
(also, if you have any suggestion to make it clearer, easier, or better performing, all are welcome)

Reply:
ah, I see. what if you try to build the absolute urls for the expected urls, then store them as `set`, then compare is against `set(executed["data"]["updateHarbor"]["harbor"]["properties"].pop("maps"))`. 🤔 might be too verbose/ugly though.

Reply:
I thought about that as well, but that would require getting the `request` fixture and building each url (`request.build_absolute_uri(get_harbor_resources_dir(...))`) and might add a bit more complexity/verbosity, but I can test it anyway.

Reply:
So, it is possible to do this, but I'm not sure if it helps ot makes it worse...

```python
request = RequestFactory().request()
expected_urls = set([
    request.build_absolute_uri(settings.MEDIA_URL + get_harbor_media_folder(harbor, file_name))
    for file_name in map_file_names
])
# Test that all the expected files are in the instance map files
assert set(map(lambda file: file["url"], map_files)) == set(expected_urls)
```

Reply:
would this work?
```python
request = RequestFactory().request()
expected_urls = set([
    request.build_absolute_uri(get_harbor_media_folder(harbor, file_name))
    for file_name in map_file_names
])
# Test that all the expected files are in the instance map files
assert set(map_files.values()) == set(expected_urls)
```

Reply:
Nope, because `map_files` is a list of `OrderedDicts` (`[{"url": "..."}]`), so you can't get values.

Reply:
ok, only loops upon loops upon loops as alternatives. let's go witht your implementation 🙂